verifier/hash_verifier.go

```diff
@@ -167,6 +171,7 @@ func (v *Verifier) verifyForSingleEvent(event *model.Event) error {
 	if challengeResErr != nil {
 		err = v.dataProvider.UpdateEventStatusVerifyResult(event.ChallengeId, model.Verified, model.HashMismatched)
 		if err != nil {
+			v.metricService.IncHashVerifierErr()
```

Comment:
Can we have another metrics to record upstream challenge api issues - i.e., the count of errors when calling the api ? Then when clearly know whether the challenge api has issue or not.

Reply:
added